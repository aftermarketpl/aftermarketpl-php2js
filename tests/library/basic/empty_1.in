$a = [];
$a[1] = 0;
return empty($a[1]);
