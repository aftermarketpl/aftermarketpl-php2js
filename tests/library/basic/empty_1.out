var a;
a = [];
a[1] = 0;
return !Boolean(a[1]);
